### ISSUE TITLE
Send Embeds not working anymore

### ISSUE THREAD

**Issue by jcalor** [Fraserbc/BetterDiscord-Embeds#41]
Hello,

Plugin is not working. When I press enter does not send anything, nor the message itself (without embed)

**Comment by D3F3ND3R1538** [comment]
same issue here

**Comment by dav1312** [comment]
Discord patched it 🤣

**Comment by sobolina420** [comment]
Having the same issues,  @dav1312 @Fraserbc  what is the solution to be made?

**Comment by ranlagunas212** [comment]
Please Kindly have this fixed as soon as possible.

**Comment by ranlagunas212** [comment]
@Fraserbc please get this fixed

**Comment by Fraserbc** [comment]
Noooooo! I've just tried some stuff and from what I can tell, discord has patched it.

**Comment by Minigodder** [comment]
yep, it's been patched by discord. beginning of the end for self-bots 😭

**Comment by Fraserbc** [comment]
I'm going to leave this open for maybe 2 days, see if anyone finds a workaround. If not, then this is the end.

**Comment by wizzzz9** [comment]
I saw that someone found solutions to the problem, but I do not know this solution :(

**Comment by Fraserbc** [comment]
Can you point me to the person who solved it? I'd like to ask them how they did it

**Comment by FSLGAMING** [comment]
![image](https://user-images.githubusercontent.com/64930236/151671974-e535bb4f-6b75-4ff8-a519-a788465f8070.png)
I found this Dm Bot (not a real Discord bot) who has sent an embed, so there has to be a workaround somehow

**Comment by dav1312** [comment]
@FSL2 They probably used links for that
![image](https://user-images.githubusercontent.com/63931154/151674460-16d98952-8128-4f3f-81d4-0a292e17fb23.png)

**Comment by FSL2** [comment]
@dav1312 Wdym? its not a picture if you mean that

**Comment by dav1312** [comment]
> @dav1312 Wdym? its not a picture if you mean that

They are probably using something like https://embed.rauf.wtf/ + an old trick to hide text inside Discord messages so the link doesn't appear

**Comment by kxacku** [comment]
> > @dav1312 Wdym? its not a picture if you mean that
> 
> They are probably using something like https://embed.rauf.wtf/ + an old trick to hide text inside Discord messages so the link doesn't appear Using this in DMs is not a huge issue, but using it in servers can get you muted or banned by a bot for spamming

how to hide the link?

**Comment by itsmeboy0** [comment]
> > @dav1312 Wdym? its not a picture if you mean that
> 
> They are probably using something like [embed.rauf.wtf](https://embed.rauf.wtf/) + an old trick to hide text inside Discord messages so the link doesn't appear Using this in DMs is not a huge issue, but using it in servers can get you muted or banned by a bot for spamming

@dav1312 Old trick? Can you elaborate? How can you send an embed without a link showing? That might be the workaround for this plugin.
